README.md

```diff
@@ -1,24 +1,65 @@
-# README
+# Budget App
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+> Description of the project.
+This project is about building a mobile web application where the user can manage their budget: they have a list of transactions associated with a category, so that they can see how much money they spent and on what.
 
-Things you may want to cover:
+I had to create a Ruby on Rails application that allows the user to:
```

Comment:
- [ ] The video should contain a highlight about a piece of the code you find interesting or are proud of making.
- [ ] you are not demonstrating the register and login part of your application which is a big part of the requirements.

Reply:
Thank you so much @mikethreels for the constructive feedback.
let me go ahead and implement those changes. :rocket: :rocket: